docs/Contributing/API-for-contributors.md

```diff
@@ -3184,6 +3185,73 @@ Notifies the server about an agent error, resulting in two outcomes:
 
 `Status: 500`
 
+### Register a device with the setup experience
+
+`POST /api/v1/fleet/device/{token}/setup_experience/start`
+
+##### Parameters
+
+| Name  | Type   | In   | Description                        |
+| ----- | ------ | ---- | ---------------------------------- |
+| token | string | path | The device's authentication token. |
+
+##### Example
+
+`POST /api/v1/fleet/device/8b49859b-1ffa-483d-ad27-85b30aa3c55f/setup_experience/start`
+
+##### Default response
+
+`Status: 201`
+
+### Get the status of a device in the setup experience
+
+`GET /api/v1/fleet/device/{token}/setup_experience/status`
```

Comment:
Heads up @noahtalerman ; we figured out that this should be an Orbit authenticated endpoint in this [slack convo](https://fleetdm.slack.com/archives/C03C41L5YEL/p1727811698445789?thread_ts=1727804597.201199&cid=C03C41L5YEL). TLDR is that we found that we can launch swiftDialog directly from Orbit, without needing to use Fleet Desktop. Making this an Orbit endpoint will make it possible to do the setup experience during macOS Setup Assistant instead of afterwards.

Reply:
FYI @marko-lisica (giving you a ping here just in case you missed this)